learning_paths/api/v1/urls.py

```diff
@@ -1,12 +1,22 @@
 """ API v1 URLs. """
 
+from django.urls import path
 from rest_framework import routers
 
-from learning_paths.api.v1.views import LearningPathAsProgramViewSet
+from learning_paths.api.v1.views import (
+    LearningPathAsProgramViewSet,
+    learning_path_progress_view,
+)
 
 router = routers.SimpleRouter()
 router.register(
     r"programs", LearningPathAsProgramViewSet, basename="learning-path-as-program"
 )
 
-urlpatterns = router.urls
+urlpatterns = router.urls + [
+    path(
+        "learning-paths/<int:learning_path_id>/progress/",
```

Comment:
Shouldn't we use UUIDs here to hide the number of existing learning paths?

Reply:
Updated it to use uuid instead.